src/Entities/Page.php

```diff
@@ -151,6 +157,16 @@ private function fillTitle(): void
         }
     }
 
+    /**
+     * 
+     */
+    private function fillUrl(): void
+    {
+        if (Arr::exists($this->responseData, 'url')) {
+            $this->url = $this->responseData['url'];
```

Comment:
rename to pageUrl? because this is not a property 🤔 + add php doc

Reply:
✅😊 done